docs/build/js-client/10-marine-js.md

```diff
@@ -30,10 +55,11 @@ const wasm = fromUint8Array(new Uint8Array(greetingWasm));
 ```
 
 
-### Registering services in FluencePeer
+### Registering service in JS client
 
-JS client loads wasm content through its own special service called `Srv`.
-You need to add the service definition if you want to use it. 
+JS client doesn't provide any specific interface for creating services in Marine JS.
+However, the client loads wasm content through its own special service called `Srv`. 
```

Comment:
is this the client or the peer?

Reply:
JS client and the peer is very similar things i believe.

Reply:
very similar or the same ? if not the same what's the difference to the (developer) user?

Reply:
The difference in very subtle. JS client is actually a JS peer wrapped in convenient API for interacting with and configuring the peer. We can assume that they are the same. I'd like to use "the client" because it's closer to repo name and actual product name "JS client".